README.md

```diff
@@ -1 +1,17 @@
-# github-status
\ No newline at end of file
+# github-status
+
+## Deployment
```

Comment:
Let's include deployment part to the previous commit.

Reply:
Fixed